src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderContactPerson.kt

```diff
@@ -35,12 +34,15 @@ data class OffenderContactPerson(
   @JoinColumn(name = "OFFENDER_BOOK_ID", nullable = false)
   val offenderBooking: OffenderBooking,
 
-  @ManyToOne(optional = false, fetch = LAZY)
+  @ManyToOne(optional = true, fetch = LAZY)
   @JoinColumn(name = "PERSON_ID")
```

Comment:
a contact is either related to a civil person or a prisoner - hence this is nullable